classnames/classnames.d.ts

```diff
@@ -6,7 +6,7 @@
 declare type ClassValue = string | number | ClassDictionary | ClassArray;
 
 interface ClassDictionary {
-	[id: string]: boolean;
+	[id: string]: any;
```

Comment:
This is what I am seeing:

![image](https://cloud.githubusercontent.com/assets/2505474/18791341/9b9ee726-81b2-11e6-9886-50693336fbef.png)

In that case I'm gonna have to use my own typings.